drivers/chibios/ws2812.c

```diff
@@ -43,11 +43,21 @@
 // These are the timing constraints taken mostly from the WS2812 datasheets
 // These are chosen to be conservative and avoid problems rather than for maximum throughput
 
+#ifndef T1H
```

Comment:
This should be done as a separate PR.  

Please pull this change out, and submit it as such.

Reply:
Done. I'll submit another PR